suite-native/accounts/src/tests/selectors.test.ts

```diff
@@ -6,6 +6,17 @@ import {
     sortAccountsByNetworksAndAccountTypes,
 } from '../utils';
 
+jest.mock('@everstake/wallet-sdk', () => ({
```

Comment:
is this needed? it is not used in suite-native at all

Reply:
otherwise, tests are failing. According to the stack trace it uses somewhere blockchain-link-utils and we have an import of  @everstake/wallet-sdk there. That's why tests are complaining.

Reply:
If it is really needed, please create a new file in `suite-native/test-utils`  please ( you can get inspired in[ this file](https://github.com/trezor/trezor-suite/blob/c82455e7465d43a24d850fef2c6e7de3f5c2c262/suite-native/test-utils/src/atomsMock.js)). Then include it in the [jest.config.native.js](https://github.com/trezor/trezor-suite/blob/5dcefbb3a030301f17a3937bac341f1e80f86773/jest.config.native.js) `setupFiles` array.

The mock should not be placed in the test source file.

Reply:
done, https://github.com/trezor/trezor-suite/pull/13267/commits/27d7c4ea17248101973f44e3194ca85000d8074e